examples/transit-gateway-cross-account-vpc-attachment/README.md

```diff
@@ -16,3 +16,7 @@ terraform apply \
 	-var="aws_second_secret_key=SuperSecretKeyForAccount2" \
 	-var="aws_region=us-east-1"
 ```
+
+## Prerequisite 
+- In the AWS console of the owner of the TGW, enabling AWS Resource Access Manager by going into `Services`, `Organization`, `Settings` and `Enable Access` to AWS Resource Access Manager. 
```

Comment:
Instead of listing out the instructions manually here, which may become out of date as the AWS console evolves, can we please point to the relevant AWS documentation instead and also note the prerequisite for the organization itself? e.g.

```markdown
## Prerequisites

- This example requires two AWS accounts within the same AWS Organizations Organization
- Ensure Resource Access Manager is enabled in your organization. For more information, see the [Resource Access Manager User Guide](https://docs.aws.amazon.com/ram/latest/userguide/getting-started-sharing.html).
```